testcase.py

```diff
@@ -644,19 +675,19 @@ def get_kpis(self):
             payload = None
             status = 500
             message = "Failed to query KPIs: {}".format(traceback.format_exc())
-            logging.warning(message)
-        logging.info(message)
+            logging.error(message)
+        logging.info(message) 
 
         return status, message, payload
 
-    def set_forecast_parameters(self,horizon,interval):
+    def set_forecast_parameters(self, horizon, interval):
         '''Sets the forecast horizon and interval, both in seconds.
 
         Parameters
         ----------
-        horizon: int
+        horizon: float
             Forecast horizon in seconds.
-        interval: int
+        interval: float
```

Comment:
@SenHuang19 A reason to change interval and horizon to float instead of int?  I think it may be safer to use int?